test/AssetHolder/claimAll.test.ts

```diff
@@ -78,26 +79,36 @@ describe('claimAll', () => {
         expect(await AssetHolder.holdings(guaranteeId)).toEqual(gHeldBefore);
       }
 
-      // compute an appropriate allocation
-      const allocation = cDestBefore.map((x, index, array) => ({
+      const allocation: Allocation = cDestBefore.map((x, index, array) => ({
         destination: x,
         amount: cAmountsBefore[index],
       }));
+      const allocationOutcome: AllocationOutcome = {
+        allocation,
+        assetHolderAddress: AddressZero,
```

Comment:
I'm not sure this belongs here. This test is targeting a specific `assetHolderAddress` by connecting to it on L27. 

I think of it this way:  The `NitroAdjudicator` contract needs to know one or more `assetHolderAddress`es to push an `AllocationOrGuarantee` to. After that point, users interact with a particular `AssetHolder` (e.g. to call `claimAll`), so there's no need to tell that contract its own address.

This has been made more confusing than it needs to be, since the aforementioned method on `NitroAdjudicator` is currently called `pushOutcome`. We should find a better name.

Reply:
You're right! I've refactored the outcome a little bit so the `encodeOutcomeContent`/`hashOutcomeContent` just takes `allocationOrGuarantee` here 25d3ed9